task_interface_mapreduce.go

```diff
@@ -0,0 +1,15 @@
+package taskgraph
+
+// MapreduceTask is a logic repersentation of a computing unit in mapreduce framework.
+// Each task contain at least one Node.
+
+type MapreduceTask interface {
```

Comment:
Don't mix user interface and framework interface.
I want you to define here the interface regarding how user can use MapReduce Framework.